firefox-ios/nimbus-features/messaging/messaging-firefox-ios.fml.yaml

```diff
@@ -37,7 +37,7 @@ import:
               ALLOWED_TIPS_NOTIFICATIONS: "allowed_tips_notifications"
 
               # Behavioral Targeting Events
-              VIEWED_HOMEPAGE:  "'homepage_viewed'|eventLastSeen('Hours') <= 24"
+              SECOND_HOMEPAGE_VIEW:  "'homepage_viewed'|eventSum('Years', 4, 0) >= 2"
```

Comment:
Your PR description says you want this to show up on the second run, does that mean second run of the app? If I run the app right now for the first time and open a second tab with home page enabled it will show the micro-survey.

Reply:
I updated the PR description to be more clear. The user should be eligible to be enrolled in the microsurvey nimbus experiment after viewing the homepage for the second time. There is also the concept of first run relating to nimbus and the microsurvey won't be a first run experiment. We haven't created the experiment yet so that testing will be done separately, but that would require a second run of the app based on this doc: https://experimenter.info/testing-on-mobile#client-side

Reply:
If you visited the homepage twice, then its expected that you will see the microsurvey prompt for this PR.

Reply:
Ahh thank you for explaining! That is helpful.